shared/teams/channel/create-channels.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as TeamsTypes from '../../constants/types/teams'
+import * as TeamsGen from '../../actions/teams-gen'
+import * as Container from '../../util/container'
+import * as Styles from '../../styles'
+import CreateChannels from '../new-team/wizard/create-channels'
+
+type Props = Container.RouteProps<{teamID: TeamsTypes.TeamID}>
+
+export default (props: Props) => {
+  const teamID = Container.getRouteProps(props, 'teamID', TeamsTypes.noTeamID)
+  const dispatch = Container.useDispatch()
+  React.useEffect(() => () => dispatch(TeamsGen.createSetChannelCreationError({error: ''})), [
+    props,
+    teamID,
+    dispatch,
+  ])
+  const [waiting, error] = Container.useSelector(s => [
```

Comment:
I think the reason your success immediately goes away is that this useSelector render thrashes since the array reference is different every time. Use 2 `useSelector`s instead to avoid it.